flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -253,7 +249,7 @@ public void checkpointState(
 		// Step (2): Send the checkpoint barrier downstream
 		operatorChain.broadcastEvent(
 			new CheckpointBarrier(metadata.getCheckpointId(), metadata.getTimestamp(), options),
-			unalignedCheckpointEnabled);
+			options.isUnalignedCheckpoint());
```

Comment:
nit: might seem more consistent if we either remove below `#includeChannelState` method or use that method in all the places.

Reply:
inlined